src/exec/Main.hs

```diff
@@ -515,7 +515,7 @@ main = do
         -- Thread registry is needed by ChainDB and by the network protocols.
         -- I assume it's supposed to be shared?
         ResourceRegistry.withRegistry $ \rr -> do
-          let protocolVersion = Cardano.ProtocolVersion 1 0 0
+          let protocolVersion = Cardano.ProtocolVersion 0 0 0
```

Comment:
My understanding of the protocol version is that there's some initial major/minor/patch versions given in some config, and that it ischanged by the blockchain update system (new major/minor/patch is put in a proposal).

So it seems blatantly wrong that it was 1.0.0 by default. Even more wrong since it should come from a configuration. But then again, the _weirdest_ thing is that the configuration gives a major/minor/patch at all! Why not set the initial version _always_ to 0.0.0 ?

Reply:
@avieth, I would agree on both counts, indeed.

I think the update system integration wasn't finalised, so that is a leftover from the initial foray into these lands.